CHANGELOG.md

```diff
@@ -30,7 +30,9 @@ bumped version for float/double range bugfix
 2. Support for monadic generators e.g. [mwc-random](https://hackage.haskell.org/package/mwc-random).
 3. Monadic adapters for pure generators (providing a uniform monadic
    interface to pure and monadic generators).
-4. Faster by more x1000 (depending on the type) - see below for benchmarks.
+4. Faster in all cases bar one by more than x18 (N.B. x18 not 18%) and
```

Comment:
What does this mean: `bar one by`? Was that intended or am I missing a piece of English vocabulary?

Reply:
bar means except probably more common in the North - the exception is Integer - I will replace bar by except

There's a famous song: On Ilkley Moor bar t'hat (on Ilkley Moor without a hat) - actually it's probably not that famous - my children wouldn't know it.